.github/workflows/code-styling.yml

```diff
@@ -0,0 +1,76 @@
+name: Code styling
+on:
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 1 * *" # first of every month at midnight
```

Comment:
I think I would not schedule this yet, and if we do, we probably want it to _not_ be the same as spelling, as then we will always have competing and potentially conflicting PRs.